src/components/common/PrivateRoute.js

```diff
@@ -4,10 +4,9 @@ import { withAuthenticationRequired } from '@auth0/auth0-react';
 import Sidebar from './Sidebar/Sidebar';
 import { Spin } from 'antd';
 
-const PrivateRoute = ({ component, path, ...args }) => (
+const PrivateRoute = ({ component, ...args }) => (
   <Sidebar>
     <Route
-      path={path}
```

Comment:
This was a good change to make the code more succinct and remove something unnecessary.